packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -106,12 +107,30 @@ export async function setupTestSuiteDatabase(
   suiteMeta: TestSuiteMeta,
   suiteConfig: NamedTestSuiteConfig,
   errors: Error[] = [],
+  alterStatement?: string,
```

Comment:
Could we have an alter statement per provider? Could be useful for testing similar things on all SQL providers

Reply:
I have made it a callback where the provider is passed in.